libs/api-skolplattformen/lib/parse/__tests__/classmates.test.ts

```diff
@@ -33,7 +33,7 @@ beforeEach(() => {
             POCODE: null,
             TELHOME: null,
             TELMOBILE: '0690-6346216',
-            EMAILHOME: 'allan.fridell@mailinater.com',
+            EMAILHOME: 'allan.fridell@mailinator.com',
```

Comment:
If this is a sample, I'd argue that it should be `@mailinator.example.com` to make that clear.